code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -1013,6 +1018,14 @@
 				abort(RITUALABORT_CONVERT)
 				message_admins("BLOODCULT: [key_name(victim)] has been converted by [key_name(converter)].")
 				log_admin("BLOODCULT: [key_name(victim)] has been converted by [key_name(converter)].")
+				if (issilicon(victim))
+					var/mob/living/silicon/S = victim
+					S.laws = new /datum/ai_laws/cultimov
```

Comment:
Isn't there a function to set the laws?

Reply:
Not that I saw, I think it's handled this way in other places (notably emagging)